helm-chart/renku/values.yaml

```diff
@@ -823,3 +853,19 @@ initDb:
   image:
     repository: renku/init-db
     tag: "latest"
+
+initRabbitmq:
+  image:
+    repository: renku/init-rabbitmq
+    tag: "latest"
+
+## Values for the built-in RabbitMQ helm chart. If an external RabbitMQ is to be
+## used then set the global.rabbitmq.enabled flag to false.
```

Comment:
Similar to what I commented above, do we need `global.rabbitmq.external.disabled` if we know `rabbitmq.enabled`?